pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -204,7 +204,8 @@ func (q *QueryReconciler) makeDispatcher(log logr.Logger, vdb *vapi.VerticaDB,
 	_ *string) (vadmin.Dispatcher, error) {
 	if vmeta.UseVClusterOps(vdb.Annotations) {
 		// The password isn't needed since our API is going to strictly communicate with the NMA
-		return vadmin.MakeVClusterOps(log, vdb, q.VRec.GetClient(), "", q.VRec, vadmin.SetupVClusterOps, q.VRec.CacheManager), nil
+		return vadmin.MakeVClusterOps(log, vdb, q.VRec.GetClient(), &vapi.EmptyPassword,
```

Comment:
Now I understand why you used a golbal variable instead of a const. But still you can just assing the empty string const to a local var and pass its pointer. It is better to avoid global vars